### ISSUE TITLE
Administration panel: add "Database" link to AppStore page

### ISSUE THREAD

**Issue by miyconst** [Starcounter/Starcounter#2798]
There should be a nice and easy way to get back to your database from AppStore page.

Currently I have to click on the *Databases* dropdown list and select my database. This requires two steps instead of one, moreover it is confusing, why should I choose my database if it is already chosen?

Things I suggest:

1. The dropdown should appear only if there are more than three databases, otherwise each database should be a "tab".
2. The database name inside the dropdown should not open the dropdown, but should open the database page. Only click on the arrow should open the drop down.
3. Add *Applications* menu item to the [SQL, AppStore, Start Executable, *] top bar menu.

**Comment by andwah** [comment]
I really like yout suggestions 1) and 2) i hope i get some time to implement them.

I think 3) is already there, it's the "House/Home" icon in the toolbar

![image](https://cloud.githubusercontent.com/assets/2656996/8540250/20a10054-2484-11e5-8ae5-06f8f5dd6681.png)

**Comment by miyconst** [comment]
This is not an urgent task, it should wait till you are back from the vacation.

3) Sorry, my bad, but could we improve the home button to look more clickable, not just an icon?

**Comment by warpech** [comment]
I agree about point 3) . The Home icon is confusing, because I would expect it to link to "Starcounter Administrator home page", not to "_default_ database home page"

I propose to follow @miyconst's advice and change the icon to a text label. I suggest "Apps" or "Control"

**Comment by miyconst** [comment]
Bump.
